content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -49,89 +38,230 @@ Click the :guilabel:`Quotation Templates` link on the :guilabel:`Settings` page,
+can immediately add them to their online sales orders themselves, if desired.
+
+.. image:: quote_template/optional-products-on-sales-order.png
    :align: center
-   :alt: Allow customers to sign electronically or to pay online on Odoo Sales.
+   :alt: Optional products appearing on a typical sales order with Odoo Sales.
 
-Design quotation templates
-==========================
+In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling product
+related to the original items in the :guilabel:`Lines` tab, if applicable. The products added here
+ideally compliment the original offering as added value for the prospective buyer.
+
+.. image:: quote_template/optional-products-tab-quotation-template.png
+   :align: center
+   :alt: Populated optional products tab on a quotation template in Odoo Sales.
+
+Clicking :guilabel:`Add a line` reveals a blank field in the :guilabel:`Product` column.
+
+When clicked, a drop-down menu with products from the database appear. Select the desired product
+from the drop-down menu to add it as an optional product to the quotation template.
+
+To delete any line item from the :guilabel:`Optional Products` tab, click the :guilabel:`🗑️ (trash
+can)` icon.
 
 .. note::
-   The following section on designing quotation templates only applies to databases that have the
-   *Quotation Builder* feature enabled, which is accessible by following the :ref:`configuration
-   steps above <sales/send_quotations/templates>`.
+   Optional products are **not** required.
+
+Terms & Conditions tab
+----------------------
+
+The :guilabel:`Terms & Conditions` tab provides the opportunity to add terms and conditions to the
+quotation template. To add terms and conditions to the quotation template, simply type (or
+copy/paste) the desired terms and conditions in this tab.
+
+.. image:: quote_template/terms-and-conditions-tab.png
+   :align: center
+   :alt: Terms and conditions tab in a quotation template form in Odoo Sales.
+
+.. seealso::
+   - :doc:`/applications/finance/accounting/customer_invoices/terms_conditions`
+
+.. note::
+   Terms and conditions are **not** required.
+
+Design quotation templates
+==========================
 
 In the upper-left corner of the quotation template form, there's a :guilabel:`Design Template`
-button. When clicked, Odoo reveals a preview of the quotation template, as it will appear on the
-front-end of the website, with a :guilabel:`Template Header` specifying that this content will
-appear on all quotations using this specific template.
+button.
+
+.. image:: quote_template/design-template-button.png
+   :align: center
+   :alt: Design template button in the upper-left corner of quotation template form.
+
+When clicked, Odoo reveals a preview of the quotation template, through the Odoo *Website*
+application, as it will appear on the front-end of the website to the customer.
```

Comment:
We may want to state explicitly somewhere in this section (maybe as a note?) that this feature is only available if you have the Website app activated